floc.bs

```diff
@@ -1 +1,149 @@
+
+  <h4 id="private-browsing-mode">Private Browsing / Incognito mode</h4>
+  In private browsing / incognito mode, using <a href="#the-api">the API</a> should throw an exception.
+
+  <h4 id="adoption-phase">Adoption phase</h4>
+  To make the adoption easier, the user agent may relax the opt-in requirement when third-party cookies still exist. For example, pages with ads resources are an approximation of the pages that are going to opt-in to interest cohort computation in the long run. Thus, at the adoption phase, the page can be eligible to be included in the interest cohort computation if there are ads resources in the page, OR if <a href="the-api">the API</a> is used.
+
+  Besides, at the adoption phase, the browser can use the existing cookie  to approximate the interest cohort permission setting. For example, when cookies are cleared, previous page visits should not be used for interest cohort computation; accessing to the interest cohort within a <a>document</a> should be denied if cookie access is not allowed in the document, or when third-party cookies are disallowed in general.
+
+  <h3 id="sensitive-information">Sensitive information</h3>
+  A cohort might reveal sensitive information. As a first mitigation, the browser should remove sensitive categories from its data collection. But this does not mean sensitive information can’t be leaked. Some people are sensitive to categories that others are not, and there is no globally accepted notion of sensitive categories.
+
+  Cohorts could be evaluated for fairness by measuring and limiting their deviation from population-level demographics with respect to the prevalence of sensitive categories, to prevent their use as proxies for a sensitive category. However, this evaluation would require knowing how many individual people in each cohort were in the sensitive categories, information which could be difficult or intrusive to obtain.
+
+  It should be clear that FLoC will never be able to prevent all misuse. There will be categories that are sensitive in contexts that weren't predicted. Beyond FLoC's technical means of preventing abuse, sites that use cohorts will need to ensure that people are treated fairly, just as they must with algorithmic decisions made based on any other data today.
+
+  <h3 id="tracking-people-via-their-interest-cohort">Tracking people via their interest cohort</h3>
+  A cohort could be used as a user identifier. It may not have enough bits of information to individually identify someone, but in combination with other information (such as an IP address), it might. One design mitigation is to ensure cohort sizes are large enough that they are not useful for tracking. The Privacy Budget explainer points towards another relevant tool that FLoC could be constrained by.
+
+  The interest cohorts over time can also be used as a user identifier. To mitigate the fingerprinting risk, the browser should ensure that the cohort does not update too often.
```

Comment:
This does not make sense to me, as a non-domain expert. Wouldn't a stable identifier be *more* useful for as a user identifier? Probably this needs some expanding.

Reply:
You are right. I updated this section.